var/spack/repos/builtin/packages/conduit/package.py

```diff
@@ -38,13 +38,14 @@ class Conduit(CMakePackage):
     git = "https://github.com/LLNL/conduit.git"
     tags = ["radiuss", "e4s"]
 
-    license("Apache-2.0")
+    license("BSD-3-Clause")
 
     version("develop", branch="develop", submodules=True)
     # note: the main branch in conduit was renamed to develop, this next entry
     # is to bridge any spack dependencies that are still using the name master
     version("master", branch="develop", submodules=True)
     # note: 2021-05-05 latest tagged release is now preferred instead of develop
+    version("0.9.3", sha256="45d5a4eccd0fc978d153d29c440c53c483b8f29dfcf78ddcc9aa15c59b257177")
```

Comment:
I'm not getting this sha256; rather `2968fa8df6e6c43800c019a008ef064ee9995dc2ff448b72dc5017c188a2e6d4`.

Reply:
Yes, sorry I messed up the paste -- that is the `0.9.2` sha.
Pushing a fix soon.